src/main/java/de/teamlapen/vampirism/config/BalanceConfig.java

```diff
@@ -330,8 +330,8 @@ public class BalanceConfig {
         vsBloodVisionDistanceSq = builder.comment("Squared blood vision distance").defineInRange("bloodVisionDistanceSq", 1600, 5, Integer.MAX_VALUE);
         vsSmallAttackDamageModifier = builder.comment("Damage added to base damage").defineInRange("smallAttackDamageModifier", 1d, 0, 10d);
         vsSmallAttackSpeedModifier = builder.comment("Basic skill - Weapon cooldown = 1/(oldvalue*(1+modifier))").defineInRange("smallAttackSpeedModifier", 0.15, 0, 3);
-        vsNeonatalReduction = builder.comment("Reduced percentage of the neonatal effect").defineInRange("neonatalReduction", 0.5, 0, 1);
-        vsDbnoReduction = builder.comment("Reduced percentage of the downed timer required to resurrect").defineInRange("dbnoReduction", 0.5, 0, 1);
+        vsNeonatalReduction = builder.comment("Reduced percentage of the neonatal effect").defineInRange("neonatalReduction", 0.5, -1024, 1024);
+        vsDbnoReduction = builder.comment("Reduced percentage of the downed timer required to resurrect").defineInRange("dbnoReduction", 0.5, -1024, 1024);
```

Comment:
What is the idea here?

The config values are applied to the attribute with `MULTIPLY_TOTAL`. It does not make sense to use values below `0` for this option

Reply:
Oops, I was not thinking at all there :P I basically just thought that since it needs to be multiplied by a high positive number to increase the value it will have to be the opposite to get really low values, so I just put the negative of the max as the min, not thinking about how that would actually work.